src/client/e2e/spot-tile.spec.ts

```diff
@@ -2,41 +2,182 @@ import { expect, Page } from "@playwright/test"
 import { test } from "./fixtures"
 
 test.describe("Spot Tile", () => {
-  test("When I sell EUR to USD then trade Id shown in tile should match trade Id shown in blotter", async ({
-    context,
-    fxOpenfinPagesRec,
-  }, testInfo) => {
-    let tilePage: Page
-    let blotterPage: Page
+  let tilePage: Page
+  let blotterPage: Page
 
+  test.beforeAll(async ({ context, fxOpenfinPagesRec }, testInfo) => {
     if (testInfo.project.name === "openfin") {
       const mainWindow = fxOpenfinPagesRec["mainWindow"]
-      // Openfin runtime is triggered inside evaluate
       await mainWindow.evaluate(async () => {
         const currentWindow = window.fin.desktop.Window.getCurrent()
         currentWindow.maximize()
         return window.fin
       })
+
       tilePage = fxOpenfinPagesRec["fx-tiles"]
       blotterPage = fxOpenfinPagesRec["fx-blotter"]
     } else {
       const pages = await context.pages()
       const mainWindow = pages.length > 0 ? pages[0] : await context.newPage()
+
       await mainWindow.goto(`${process.env.URL_PATH}`)
+
       tilePage = mainWindow
       blotterPage = mainWindow
     }
+  })
+
+  test.describe("Valid Purchase", () => {
```

Comment:
In time, I think we will need to find a way to split these into _smoke_ tests and _regression_ tests ..
When I worked with Selenium way back in the dark(er) ages .. there was a way of 'tagging' the tests so one could run them by tag instead of all-or-nothing, so a select few were tagged as `@smoke` and we managed the different types of test run that way ...

Reply:
Cool, I had no idea. It is good to split them by tagging. I will investigate how to do that with playwright.